invenio_rdm_records/fixtures/vocabularies.py

```diff
@@ -344,9 +348,9 @@ def create_record(self, data, delay=False):
 class GenericVocabularyEntry(VocabularyEntry):
     """Vocabulary fixture with single data-file."""
 
-    def __init__(self, directory, id_, entry):
+    def __init__(self, directory, id_, entry, service_str="vocabulary_service"):
```

Comment:
This is not the most beautiful change ever, but:
- is back compat
- allows usage of custom service for vocabularies without schemes

Alternative: move all templated methods to `VocabularyEntry`. This would be better, but since we are moving to DataStreams soon, maybe is better not to make big changes here.